eng/common/TestResources/New-TestResources.ps1

```diff
@@ -140,6 +140,7 @@ function Retry([scriptblock] $Action, [int] $Attempts = 5)
                 Start-Sleep -Seconds $sleep
             } else {
                 Write-Error -ErrorRecord $_
+                throw
```

Comment:
This ends up adding two records to the `$Error` stream. Better to just `throw $_`, which will also effectively `write-error` anyway.